src/main/java/seedu/duke/Storage.java

```diff
@@ -116,124 +132,118 @@ public boolean checkPair() {
      * @param hasPropertyFile boolean value on whether property text file exist
      * @param hasClientFile boolean value on whether client text file exist
      * @param hasPairingFile boolean value on whether pairing text file exist
-     * @param clientList the array list containing the list of client
-     * @param propertyList the array list containing the list of property
-     * @param pairingList the hash map containing the pairing between client and property
      */
     public void loadFiles(boolean hasDirectory, boolean hasPropertyFile, boolean hasClientFile,
-                          boolean hasPairingFile, ClientList clientList, PropertyList propertyList,
-                          PairingList pairingList) {
+                          boolean hasPairingFile) {
         if (!hasDirectory) {
             makeDirectory();
         }
 
         if (hasClientFile) {
             File clientFile = new File(CLIENT_PATH);
             assert clientFile.exists() : "Client text file does not exist";
-            loadClient(clientList, clientFile);
+            loadClient(clientFile);
         }
 
         if (hasPropertyFile) {
             File propertyFile = new File(PROPERTY_PATH);
             assert propertyFile.exists() : "Property text file does not exist";
-            loadProperty(propertyList, propertyFile);
+            loadProperty(propertyFile);
         }
 
         if (hasPairingFile) {
             File pairingFile = new File(PAIR_PATH);
             assert pairingFile.exists() : "Pairing text file does not exist";
-            loadPair(pairingList, pairingFile);
+            loadPair(pairingFile);
         }
     }
 
     /**
      * Adds the client list in the text file to the array list.
      *
-     * @param clientList Client List object that contains client's array list
      * @param clientFile The file that stores the list of client.
      */
-    public void loadClient(ClientList clientList, File clientFile) {
-
+    public void loadClient(File clientFile) {
         try {
             Scanner scanner = new Scanner(clientFile);
             while (scanner.hasNext()) {
                 String[] clientParameters = scanner.nextLine().split("\\s\\|\\s");
-                String clientName = clientParameters[0];
-                String clientContact = clientParameters[1];
-                String clientEmail = clientParameters[2];
-                String clientBudget = clientParameters[3].trim();
-                clientList.addClient(clientName, clientContact, clientEmail, clientBudget);
+                if (clientParameters.length == 4) {
```

Comment:
You could avoid the use of magic literals by creating constants for the positions of the data